src/signals/incident/components/form/FileInput/index.js

```diff
@@ -8,6 +10,12 @@ import './style.scss';
 
 export const ERROR_TIMEOUT_INTERVAL = 8000;
 
+const FileInputError = styled.div`
+    color: ${themeColor('secondary')};
+    margin: 16px 0 0;
```

Comment:
Shouldn't we be using `themeSpacing` here?

Reply:
Indeed! Thanks!